app/actors/hyrax/actors/model_actor.rb

```diff
@@ -4,21 +4,15 @@ module Actors
     class ModelActor < AbstractActor
       # @param [Hyrax::Actors::Environment] env
       # @return [Boolean] true if update was successful
-      def update(env)
-        model_actor(env).update(env)
-      end
+      delegate :update, to: :model_actor
```

Comment:
Maybe I'm tired, but how does this work? How does model actor get it's parameter passed to it?

Reply:
huh. I think this was a RuboCop autocorrect. I wonder if this might be a bug in the autocorrect code. If it passes CI, I'll look into it. If it fails, I'll revert the change and fence the rule.

Reply:
Thanks for the thorough review.

Reply:
Sure enough: https://github.com/bbatsov/rubocop/issues/5393